infrastructure/main.tf

```diff
@@ -13,3 +13,60 @@ locals {
   prefix            = "${var.product}-ss"
   prefix_no_special = replace(local.prefix, "-", "")
 }
+
+# Resource Group
+
+resource "azurerm_resource_group" "rg" {
+  name     = "darts-api-dev"
+  location = "uksouth"
+  #tags     = var.resource_group_tags
+}
+
+# Application Insights
+
+resource "azurerm_application_insights" "appinsights" {
+  name                = "darts-api-appinsights-dev"
+  location            = azurerm_resource_group.rg.location
+  resource_group_name = azurerm_resource_group.rg.name
+  application_type    = "web"
+  #tags                = var.common_tags
+
+  lifecycle {
+    ignore_changes = [
+      # Ignore changes to appinsights as otherwise upgrading to the Azure provider 2.x
+      # destroys and re-creates this appinsights instance
+      application_type,
+    ]
+  }
+}
+
+# this key vault is created in every environment, but preview, being short-lived,
+# will use the aat one instead
+# This section includes hardcoded variables that were in variables.tf ...
+# ... said variables were moved here due to a Jenkins objection
+module "key-vault" {
```

Comment:
Please follow the pattern on here: https://github.com/hmcts/cnp-module-key-vault/tree/master#usage